Backpack/src/main/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetBehaviour.kt

```diff
@@ -0,0 +1,33 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.bottomsheet
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+
+class BpkBottomSheetBehaviour<V : View> : BottomSheetBehavior<V> {
```

Comment:
The swiping behaviour we need is implemented in BottomSheetBehavior. We only extends it to have custom logic entrypoint here for the future.